eng/Build.props

```diff
@@ -5,6 +5,7 @@
     <ProjectToBuild Include="$(MSBuildThisFileDirectory)..\template_feed\NetCoreTestTemplates.sln" />
     <ProjectToBuild Include="$(MSBuildThisFileDirectory)..\Templates\MSTestTemplates.sln" Condition="'$(DotNetBuildSourceOnly)' != 'true'" />
     <ProjectToBuild Include="$(MSBuildThisFileDirectory)..\WizardExtensions\WizardExtensions.sln" Condition="'$(DotNetBuildSourceOnly)' != 'true'" />
-    <ProjectToBuild Include="$(MSBuildThisFileDirectory)..\test\Microsoft.TestTemplates.Acceptance.Tests.sln" Condition="'$(DotNetBuildSourceOnly)' != 'true'" />
+    <!-- Contains only a single test project. Exclude if not building tests. Otherwise nuget will fail to find a project to restore -->
+    <ProjectToBuild Include="$(MSBuildThisFileDirectory)..\test\Microsoft.TestTemplates.Acceptance.Tests.sln" Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(DotNetBuildSkipTests)' != 'true'" />
```

Comment:
I would remove the `DotNetBuildSourceOnly != 'true'` condition. That's already the default when building from source. If someone wants to build test-templates with tests when building from source, they should be able to.